packages/utils/src/lib/logger.ts

```diff
@@ -532,3 +532,42 @@ export class Logger {
  * logger.info('Made with ❤️ by Code PushUp');
  */
 export const logger = new Logger();
+
+/**
+ * Formats a command string for display with status indicator.
+ *
+ * @param bin Command string with arguments.
+ * @param options Command options (cwd, env).
+ * @param status Command status ('pending' | 'success' | 'failure').
+ * @returns Formatted command string with colored status indicator.
+ */
+export function formatCommand(
```

Comment:
How about `formatCommandStatus`? The name `formatCommand` is a little ambiguous. Initially, I thought it was the `[command, ...args ?? []].join(' ')` logic.